staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -1531,7 +1534,8 @@ properties:
 			if len(tst.expectedViolations) == 0 {
 				// wait for condition to not appear
 				var cond *apiextensionsv1.CustomResourceDefinitionCondition
-				err = wait.PollUntilContextTimeout(context.Background(), 100*time.Millisecond, 5*time.Second, true, func(ctx context.Context) (done bool, err error) {
+				err = wait.PollUntilContextTimeout(context.Background(), 100*time.Millisecond, 5*time.Second, true, func(ctx context.Context) (bool, error) {
```

Comment:
nit: would redefine err here through `err := ` because the scope is different to the err you are using now.